dashboard/UI-Test/README.md

```diff
@@ -0,0 +1,92 @@
+[WTRobot](https://pypi.org/project/wtrobot/) is a keyword driven UI/frontend testing framework.
+
+## Setup
+
+### Install
+
+```console
+
+> pip install wtrobot
+
+```
+
+> NOTE
+
+- Selenium_drivers folders have your selenium webdrivers geckodrivers(for firefox) and chromedrivers(for chrome and chromium)
+
+- If script fails due to drivers issue, you need to find appropriate selenium webdriver according to your browser version
+
+  - [firefox](https://github.com/mozilla/geckodriver/releases) & [chrome/chromium](https://chromedriver.chromium.org/downloads)
+
+- Unzip or untar the executable and place in selenium_drivers dir.
+
+## Executing Script
+
+- Write all your test cases into test.yaml and execute
+
+```console
+
+> wtrobot
+
+```
+
+> NOTE
+
+- If config(in `config.json`) is missing on initial run, tool will ask you for few configuration question and create `config.json` file.
```

Comment:
I think the phrase, "ask you for few configuration question", should be "ask you a few configuration questions".